### PR TITLE
Study View: Restrict api calls if there are no valid studies in query

### PR DESCRIPTION
**Pull request by kalletlak**
Fix https://github.com/cBioPortal/cbioportal/issues/7381

### CODE REVIEW

**Comment by alisman** [comment]
What UI shows up in this case?   Do we have any idea how these invalid study ids came about?  
We might want to handle this with global error handler.   Lets discuss.

**Comment by kalletlak** [comment]
It might be unauthorized studies. I don't have access to this study.

Screen Shot 2020-04-13 at 1 25 18 PM

File: src/pages/studyView/StudyViewPage.tsx [hunk 1]
```diff
@@ -386,7 +386,9 @@ export default class StudyViewPage extends React.Component<
                 {this.store.queriedSampleIdentifiers.isComplete &&
                     this.store.invalidSampleIds.isComplete &&
                     this.store.unknownQueriedIds.isComplete &&
-                    this.store.displayedStudies.isComplete && (
+                    this.store.displayedStudies.isComplete &&
+                    this.store.queriedPhysicalStudies.isComplete &&
```
Comment (alisman) [inline src/pages/studyView/StudyViewPage.tsx:390]: i'm curious to know why this wouldn't be caught by unknownQueriedIds (above).  Maybe the study list service returns the ID even if user doesn't have access to it?

**Comment by alisman** [comment]
I think this is ok.  Would sort of be nice to have more explicit validation routine so that reader can understand exactly what needs to be true for studies to be valid.

**Pull request merged**